### PR TITLE
[DEBUGGER] Add span and span-decoration method probe snapshot tests

### PR DESCRIPTION
**Pull request by shurivich**
## Description
Add span and span-decoration method probe snapshot tests

### CODE REVIEW

File: tests/debugger/test_debugger.py [hunk 1]
```diff
@@ -155,19 +183,37 @@ def wait_for_probe(self, data):
...
+        self.span_decoration_probe_response = weblog.get("/debugger/span-decoration/asd/1")
 
     def test_method_probe_snaphots(self):
         assert self.remote_config_is_sent is True
         assert self.probe_installed is True
-        assert self.log_probe_response.status_code == 200
 
-        expected_data = ["logProbe-installed"]
-        validate_data(expected_data, expected_data)
+        assert self.log_probe_response.status_code is 200
```
Comment (cbeauchesne) [inline tests/debugger/test_debugger.py:202]: Even if it works, `is` must be reserved for reference comparizon. See the output form the interpreter : 

image


```suggestion
        assert self.log_probe_response.status_code == 200
```

File: tests/debugger/test_debugger.py [hunk 2]
```diff
@@ -35,13 +39,32 @@ def get_debugger_map():
...
+                    for span in chunk["spans"]:
+                        if span["name"] == "dd.dynamic.span":
+                            span_hash[span["meta"]["debugger.probeid"]] = span
+                        else:
+                            for key, value in span["meta"].items():
+                                if key.startswith("_dd.di"):
+                                    span_hash[value] = span["meta"][key.split(".")[2]]
 
-    return hash
+    print(span_hash)
```
Comment (cbeauchesne) [inline tests/debugger/test_debugger.py:66]: Could you remove the print ?

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: tests/debugger/test_debugger.py (hunk 1 above)
Comment (shurivich) [reply tests/debugger/test_debugger.py]: Fixed here: de6a537028ac016bb10d972b0903b79c9c02bf26

File: tests/debugger/test_debugger.py (hunk 2 above)
Comment (shurivich) [reply tests/debugger/test_debugger.py]: Fixed here de6a537028ac016bb10d972b0903b79c9c02bf26

File: tests/debugger/test_debugger.py (hunk 1 above)
Comment (shurivich) [reply tests/debugger/test_debugger.py]: I changed back to `is True` here a04a833aa5bd8caaf3c415187c00b2f6648cdf1d

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**